langkit/compile_context.py

```diff
@@ -1403,6 +1422,14 @@ def emit(self, file_root='.', main_source_dirs=set(), main_programs=set(),
         :param bool generate_gdb_hook: Whether to generate the
             ".debug_gdb_scripts" section. Good for debugging, but better to
             disable for releases.
+
+        :param list[str] plugin_passes: List of passes to add as plugins to the
+            compilation pass manager. List item must be a name matching the
+            following pattern: ``MODULE.CALLABLE`` where ``MODULE`` is the name
```

Comment:
Why u CAP ?

Reply:
I generally use upper case for pattern chunks and lower case for the rest.